convert2rhel/systeminfo.py

```diff
@@ -77,6 +79,8 @@ def __init__(self):
         self.distribution_id = None
         # Major and minor version of the operating system (e.g. version.major == 8, version.minor == 7)
         self.version = None
+        # String of the system version, either joined major and minor version (8.5, 9.4) or just the major version
+        self.version_str = None
```

Comment:
We should stick to one value, Could set the minor version to None and instead have a `getVersion()` function that returns a string

So `self.version` could be `{major: "8", minor: None}`

Reply:
I don't think I follow @Venefilyn, really.
`self.version` still is a tuple, and we can access `self.version.minor` and `self.version.major` if ever needed.
We're working with the version string, where we just log the value.
But I understand the benefit of having it returned by a function.

Reply:
Yes, that is what I want it to remain, but having a separate string like this is IMO not a good solution, I'd rather want to see a separate `getVersion()` or just changing the version nameddict to handle `__repr__` so it can be put into a string easily

Reply:
I like the `__repr__` method approach, let me try that